pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://pwa.ir
 repository: https://github.com/xclud/flutter_map
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
```

Comment:
The same here.

Reply:
I think the the correct approach is with -0  
See migration and (un-)sound null safety here: https://dart.dev/null-safety/unsound-null-safety
But it's your project. And Flutter 2 is in stable today :-)

Reply:
Right. Flutter 2 and Dart 2.12 are out now officially. We can go for 2.12.0 now,

And no... it's not my project. It's open source and belongs to you as well :)

Reply:
The link you've posted is now updated and has removed the `-0` part.